### PR TITLE
Adding borrowing model, serializers for this model, and 2 views for B…

### PR DESCRIPTION
**Pull request by DmytryTong**
…orrowing list and detail. Also register model in a admin.py

### CODE REVIEW

File: borrowing/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+
+class Borrowing(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
```
Comment (oksana0410) [inline borrowing/models.py:6]: add related_name

File: borrowing/models.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```
Comment (oksana0410) [inline borrowing/models.py:7]: add related_name

File: borrowing/serializers.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    class Meta:
+        model = Borrowing
+        fields = (
+            'id',
+            'book',
+            'user',
+            'borrow_date',
+            'expected_return_date',
+            'actual_return_date'
+        )
```
Comment (yura-hudzovskyi) [inline borrowing/serializers.py:18]: Fix quotes

File: borrowing/models.py (hunk 2 above)
Comment (yura-hudzovskyi) [inline borrowing/models.py:7]: use settings.AUTH_USER_MODEL

**Review by yura-hudzovskyi — CHANGES_REQUESTED** [review]

File: borrowing/models.py (hunk 1 above)
Comment (DmytryTong) [reply borrowing/models.py:6]: Fixed

File: borrowing/models.py (hunk 2 above)
Comment (DmytryTong) [reply borrowing/models.py:7]: Fixed

File: borrowing/serializers.py (hunk 3 above)
Comment (DmytryTong) [reply borrowing/serializers.py:18]: Fixed.

File: borrowing/models.py (hunk 2 above)
Comment (DmytryTong) [reply borrowing/models.py:7]: Fxed.

**Review by oksana0410 — APPROVED** [review]

**Review by yura-hudzovskyi — APPROVED** [review]

**Pull request merged**